frontend/seo/matrix.ts

```diff
@@ -0,0 +1,43 @@
+import { useDBInstanceStore } from "../src/stores/dbInstance";
+import { CloudProvider } from "../src/types";
+
+export const cloudProviderList: string[] = ["AWS", "GCP"];
+
+export const cloudServiceList: (cloudProvider: CloudProvider) => string[] = (
+  cloudProvider: CloudProvider
+) => {
+  if (cloudProvider === "AWS") {
+    return ["RDS"];
+  } else if (cloudProvider === "GCP") {
+    return ["Cloud SQL"];
+  }
+  return [];
+};
+
+export const databaseList: string[] = ["MySQL", "PostgreSQL"];
+
+// charge type On Demand is set to "", for user may not type in On Demand explicitly.
+export const chargeTypeList: string[] = ["Reserved", ""];
+
+export const machineTypeList: (cloudProvider: CloudProvider) => string[] = (
+  cloudProvider: CloudProvider
+) => {
+  const dbInstanceStore = useDBInstanceStore();
+  dbInstanceStore.loadDBInstanceList();
+  const dbInstance = dbInstanceStore.dbInstanceList;
+
+  const _machineTypeList: string[] = [];
+  dbInstance.forEach((instance) => {
+    if (instance.cloudProvider === cloudProvider) {
+      if (cloudProvider === "GCP") {
+        // e.g. db-N1Standard-96-360 -> N1Standard
```

Comment:
use lowercase and either keep db prefix or remove it for all cloud providers.

Reply:
Should we apply lower case for all the params in URL?

Reply:
SG